src/torchmetrics/aggregation.py

```diff
@@ -348,14 +348,7 @@ def update(self, value: Union[float, Tensor], weight: Union[float, Tensor] = 1.0
         if value.numel() == 0:
             return
         # broadcast weight to value shape
-        if hasattr(torch, "broadcast_to"):
-            weight = torch.broadcast_to(weight, value.shape)
-        else:
-            if weight.shape == ():
-                weight = torch.ones_like(value) * weight
-            if weight.shape != value.shape:
-                raise ValueError("Broadcasting not supported on PyTorch <1.8")
-
+        weight = torch.broadcast_to(weight, value.shape)
```

Comment:
this does not look like docs chnage

Reply:
true, just a small change that was missing from #1263